configure.php

```diff
@@ -141,19 +141,119 @@ function replaceForAllOtherOSes(): array
     return explode(PHP_EOL, run('grep -E -r -l -i ":author|:vendor|:package|VendorName|skeleton|migration_table_name|vendor_name|vendor_slug|author@domain.com" --exclude-dir=vendor ./* ./.github/* | grep -v '.basename(__FILE__)));
 }
 
+function getGithubApiEndpoint(string $endpoint): ?stdClass
```

Comment:
Could you use the correct casing: `GitHub`?

Reply:
@freekmurze Absolutely, thanks for catching that. Done.